src/cli-validator/index.js

```diff
@@ -38,6 +38,10 @@ program
   )
   .option(
     '-c, --config <file>', 'path to config file, used instead of .validaterc if provided'
+  )
+  .option(
+    '-e, --errors_only',
+    'Only print the errors, ignore the warnings.'
```

Comment:
To be consistent, this should start with a lowercase letter and not end with a period